wix/windows-connector.wxs

```diff
@@ -75,12 +75,14 @@
           Vital="yes"
           Arguments='--config-filename="[CloudPrintDataFolder]gcp-windows-connector.config.json"'>
         <ServiceDependency Id="Spooler" />
+        <ServiceDependency Id="Tcpip" />
       </ServiceInstall>
       <ServiceControl
           Id="StartAndStopService"
           Name="Cloud Print Connector"
           Remove="uninstall"
           Start="install"
+          Wait="no"
```

Comment:
I don't like the idea of not waiting, because it ends up ignoring errors.
During uninstall, it ends up causing the installer to continue, and thus the service may be running leaving files in use during DeleteFiles, and potentially causes DeleteServices to leave the service behind as well.

I think a better way to do what you want (skip starting the service if you didn't run init) is to create a separate component with a different ServiceControl element in it that has the Start="install" and remove Start from this one.  Set the component condition on that other component to:
CONFIGFILE OR RUNINIT="YES"

Or maybe use a separate property entirely.

Does that sound reasonable?